packages/wdio-cli/src/launcher.js

```diff
@@ -24,6 +25,13 @@ class Launcher {
         const specs = this.configParser.getSpecs()
 
         if (config.outputDir) {
+            /**
+             * Create the outputDir if doesn't exist
+             */
+            if (!fs.existsSync(path.join(config.outputDir))) {
+                log.info('Creating ' + config.outputDir + ' directory')
```

Comment:
It might be clearer to use string template? 
```js
log.info(`Creating ${config.outputDir} directory`)
```

Reply:
Our just 'creating directory', condigt.outputDir

Reply:
I will update that

Reply:
Updated